src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests/EventHandlers/CohortAssignedToProviderEventHandlerTests.cs

```diff
@@ -63,7 +72,7 @@ public CohortAssignedToProviderEventHandlerTestsFixture() : base((m) => null)
         public CohortAssignedToProviderEventHandlerTestsFixture SetupNonTransferCohort()
         {
             GetCohortSummaryQueryResult = DataFixture.Build<GetCohortSummaryQueryResult>()
-                .With(p => p.CohortId, Message.CohortId).Without(p => p.TransferSenderId).Create();
+                .With(p => p.CohortId, Message.CohortId).Without(p => p.TransferSenderId).Without(p => p.ChangeOfPartyRequestId).Create();
```

Comment:
Why is this tests chagned, probably for a good reason, but just want to understand.